crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -854,7 +849,7 @@ mod tests {
                     "backoffMinMs": "3500",
                 },
                 "tx5Transport": {
-                    "timeoutS": "10",
+                    "signalAllowPlainText": "true"
```

Comment:
Now that timeoutS is automatically overwritten, we need to use a different field here to test that it is preserved.